src/main/java/org/wso2/carbon/module/swiftiso20022/constants/MT940Constants.java

```diff
@@ -27,4 +27,84 @@ public class MT940Constants {
+    public static final String DN_ACC_IDENTIFICATION = "Account Identification";
+    public static final String DN_STATEMENT_NUMBER = "Statement Number";
+    public static final String DN_OPENING_BALANCE = "Opening Balance";
+    public static final String DN_TRANSACTION = "Transaction";
+    public static final String DN_AMOUNT = " Amount";
+    public static final String DN_CURRENCY = " Currency";
+    public static final String DN_DATE = " Date";
+    public static final String DN_INDICATOR = " Indicator";
+    public static final String DN_STATEMENT_TYPE = " Statement Type";
+    public static final String DN_CLOSING_BALANCE = "Closing Balance";
+    public static final String DN_CLOSING_AVAIL_BALANCE = "Closing Available Balance";
+    public static final String DN_FORWARD_CLOSING_AVAIL_BALANCE = "Forward Available Balance";
+    public static final String DN_STATEMENT_LINE = "Statement Line";
+    public static final String DN_CUSTOMER_REFERENCE = "Customer Reference";
+    public static final String DN_TRANSACTION_REFERENCE = "Transaction Reference";
+    public static final String DN_TRANSACTION_TYPE = "Transaction Type";
+    public static final String DN_TRANSACTION_IND = "Transaction Indicator";
+
+    //MT940 payload fields
+    public static final String HEADER_BLOCK_1 = "block1";
+    public static final String HEADER_BLOCK_2 = "block2";
+    public static final String HEADER_BLOCK_3 = "block3";
+    public static final String REFERENCE = "reference";
+    public static final String SEQUENCE_NO = "sequenceNumber";
+    public static final String ACC_NUMBER = "accountNumber";
+    public static final String ACC_NUMBER_IDENTIFICATION = "accountNumberIdentifier";
+    public static final String OPENING_BAL_DETAILS = "openingBalanceDetails";
+    public static final String CLOSING_BALANCE_DETAILS = "closingBalanceDetails";
+    public static final String CLOSING_AVAIL_BALANCE_DETAILS = "closingAvailableBalanceDetails";
+    public static final String FORWARD_CLOSING_AVAIL_BALANCE_DETAILS = "forwardAvailableBalanceDetails";
+    public static final String TRANSACTIONS = "transactions";
+    public static final String BAL_AMOUNT = "balanceAmount";
+    public static final String BAL_CURRENCY = "currency";
+    public static final String BAL_DATE = "date";
+    public static final String BAL_INDICATOR = "indicator";
+    public static final String BAL_STATEMENT_TYPE = "statementType";
+    public static final String TRANSACTION_AMOUNT = "amount";
+    public static final String TRANSACTION_CURRENCY = "currency";
+    public static final String TRANSACTION_DATE = "dateTime";
+    public static final String TRANSACTION_INDICATOR = "transactionIndicator";
+    public static final String CUSTOMER_REFERENCE = "customerReference";
+    public static final String TRANSACTION_REFERENCE = "transactionReference";
+    public static final String TRANSACTION_TYPE = "transactionType";
+    public static final String TRANSACTION = "transaction";
+
+
```

Comment:
```suggestion

```

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/6/commits/8076e9b5033a9608b6839de38320ec994e01055d